frontends/web/src/components/guide/guide.tsx

```diff
@@ -54,11 +59,35 @@ const Guide = ({ children, title = t('guide.title') }: TProps) => {
         <div className={style.content}>
           {children}
           <div className={style.appendix}>
-            {t('guide.appendix.text')}
-            {' '}
-            <A className={style.link} href="https://bitbox.swiss/support/">
-              {t('guide.appendix.link')}
-            </A>
+            {service ? (
+              <>
+                <span className={style.question}>{t('guide.appendix.questionService', { serviceName: service.name })}</span>
+                <br />
+                <br />
+                {t('guide.appendix.textService', { serviceName: service.name })}
+                {' '}
+                {service.linkDisplay.includes('@') ? (
```

Comment:
nit: in theory uri's could contain `@` symbol as well.

With that in mind an alternative may be to make link optional  `link?: string;` (as you don't use service.link in this case).